anchantm.json

```diff
@@ -0,0 +1,15 @@
+{
+"firstName": "Mayte",
+"lastName": "Anchante",
+"nickName": "",
+"privateName": true,
+"picture": https://github.com/DH2019/assignment-00-class-profile-anchantm/blob/master/IMG_6626.JPG?raw=true
```

Comment:
this needs to be enclosed in double quotes! But also, you are going to need to change the value.  

The right thing to do is: 
- rename the image "anchantm.jpg"
- change this line to: `"picture": "images/anchantm.jpg",`
- commit your changes
- push your changes.
At that point you should be ready to pass the tests.  However, you may want ot run `npm test` first.